command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@wking said: `I don't think we want humans reading code to be part of certification,
and am perfectly happy certifying a shim/core system as a single black
box entity [1].  Folks interested in consuming the certified runtime
should use the same shim/core system that was certfied. ` 

Heh, no offense but I don't trust black boxes ;-)  My point was "if" someone wanted to certify a product with a different type of interface "then" we could certify that product against the runtime spec after inspecting the shim.  I'm not saying we "have" to I'm saying it's a viable point of discussion.